android/app/build.gradle

```diff
@@ -339,18 +344,6 @@ android {
     sourceCompatibility JavaVersion.VERSION_17
     targetCompatibility JavaVersion.VERSION_17
   }
-  packagingOptions.jniLibs {
-    excludes += [
-        'lib/**/libVkLayer_khronos_validation.so',
-        'lib/**/libVkLayer_core_validation.so',
-        'lib/**/libVkLayer_threading.so',
```

Comment:
These files are downloaded by a script, right? Does it make sense to always exclude them from Release builds?

How slow is the debug build with validation enabled?

Reply:
Validation should not be published by performance reason, it's slow

Reply:
Excluded for release

Reply:
I'm not an expert in gradle, but it looks like this parameter can't be set for release only. I tried, but it affects on all the flavours.

Reply:
Maybe it can be solved using flavors (e.g. a flavor with validation can be added and both debug and release builds can be tested). We can leave it for later.